src/Util/Frame.php

```diff
@@ -49,7 +60,7 @@ public function getFile()
     }
 
     /**
-     * @return int|null
+     * @return integer|null
```

Comment:
> I agree. I use the full name myself.

It's standard across the league repos to use the "correct" types. "integer" is only an alias of "int". It's possible hat php 7 will drop support for things like is_integer and you'll have to use is_int. My point is that "int" is the only correct thing to use here, so it's not actually a personal preference. "integer" is just wrong.